paasta_tools/frameworks/native_scheduler.py

```diff
@@ -78,7 +88,7 @@ def __init__(self, service_name, instance_name, cluster,
                  system_paasta_config, soa_dir=DEFAULT_SOA_DIR,
                  service_config=None, reconcile_backoff=30,
                  instance_type='paasta_native', service_config_overrides={},
-                 reconcile_start_time=float('inf')):
+                 reconcile_start_time=float('inf'), constraints=[]):
```

Comment:
crap, i just had a conversation about this couple days ago :) ruby habits will be dying slowly. thanks!